sympy/printing/tests/test_latex.py

```diff
@@ -141,6 +141,9 @@ def test_latex_basic():
     p = Symbol('p', positive=True)
     assert latex(exp(-p)*log(p)) == r"e^{- p} \log{\left (p \right )}"
 
+    assert latex(log(x)+log(y)) == r"\log{\left (x \right )} + \log{\left (y \right )}"
```

Comment:
may be put these tests in a new function `test_latex_log`? Since `test_latex_basic` doesn't seem like the best to me.